code/modules/atmospherics/gasmixtures/reaction_factors.dm

```diff
@@ -100,13 +101,13 @@
 		/datum/gas/tritium = "20 moles of tritium needs to be present for the reaction to occur. Tritium is consumed at 1 reaction rate",
 		/datum/gas/nitrogen = "10 moles of tritium needs to be present for the reaction to occur. Nitrogen is consumed at 1 reaction rate",
 		/datum/gas/nitrium = "Nitrium is produced at 1 reaction rate",
-		"Temperature" = "Can only occur above [NITRIUM_FORMATION_MIN_TEMP] kelvins",
+		"Temperature" = "Can only occur above [NITRIUM_FORMATION_MIN_TEMP] kelvins. Higher temperature increases the reaction rate.",
 		"Energy" = "[NITRIUM_FORMATION_ENERGY] joules of energy is absorbed per reaction rate",
 	)
 
 /datum/gas_reaction/nitrium_decomposition/init_factors()
 	factor = list(
-		/datum/gas/oxygen = "[MINIMUM_MOLE_COUNT] moles of oxygen need to be present for the reaction to occur. Not consumed.",
+		/datum/gas/oxygen = "[MINIMUM_MOLE_COUNT] moles of oxygen need to be present for the reaction to occur. Not consumed. Increases reaction rate.",
 		/datum/gas/nitrium = "Nitrium is consumed at 1 reaction rate",
 		/datum/gas/hydrogen = "Hydrogen is produced at 1 reaction rate",
 		/datum/gas/nitrogen = "Nitrogen is produced at 1 reaction rate",
```

Comment:
imo we prob shouldn't be mentioning reaction rate as a unit in our uis like this, but that can get handled later

Reply:
I suppose it could be something like "X is produced per mol of Y consumed" or something like that. Probably out of scope for this PR though.